inference-engine/tools/compile_tool/main.cpp

```diff
@@ -327,6 +330,19 @@ static void setDefaultIOPrecisions(InferenceEngine::CNNNetwork &network) {
             }
         }
     }
+
+    if (isVPUX) {
+        const InferenceEngine::Precision input = InferenceEngine::Precision::U8;
+        const InferenceEngine::Precision output = InferenceEngine::Precision::FP32;
+
+        for (auto &&layer : network.getInputsInfo()) {
+            layer.second->setPrecision(input);
```

Comment:
Please add `if (isFP32(layer.second->getPrecision())) {` check (in the same way as in `isMyriad` case). Some networks has INT64 inputs/outputs, which can't be switch to U8/FP16/FP32.

Reply:
done